cloud/amazon/ec2_vpc_route_table.py

```diff
@@ -465,17 +471,19 @@ def create_route_spec(connection, routes, vpc_id):
 def ensure_route_table_present(connection, module):
 
     lookup = module.params.get('lookup')
-    propagating_vgw_ids = module.params.get('propagating_vgw_ids', [])
+    propagating_vgw_ids = module.params.get('propagating_vgw_ids')
```

Comment:
`module.params.get` only takes one argument.  Any default should be set via module spec or python expression.